mandatory/2-function-creation.js

```diff
@@ -25,7 +30,11 @@ Write a function that:
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+     return arrayOfStrings.trim().replace("/","") .toLowercase() ; 
```

Comment:
Great start to this one, this line of code is completely correct! There are a couple more bits to add, have a read of the question again and see what the input on the function is (is it just one string?)

Reply:
can you explain it more? I really dont know what you meant and what shall I add .